agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -478,19 +478,17 @@ public Builder setAadAuthentication(Configuration.AadAuthentication aadAuthentic
       return this;
     }
 
-    public Builder setConnectionStrings(
-        @Nullable String connectionString,
-        @Nullable String statsbeatInstrumentationKey,
-        @Nullable String statsbeatEndpoint) {
-
+    public Builder setConnectionStrings(@Nullable String connectionString) {
       if (Strings.isNullOrEmpty(connectionString)) {
         this.connectionString = null;
         this.statsbeatConnectionString = null;
       } else {
         this.connectionString = ConnectionString.parse(connectionString);
         this.statsbeatConnectionString =
             StatsbeatConnectionString.create(
-                this.connectionString, statsbeatInstrumentationKey, statsbeatEndpoint);
+                this.connectionString,
+                System.getProperty("statsbeat.ikey"),
+                System.getProperty("statsbeat.endpoint"));
```

Comment:
we have a couple system properties already under `applicationinsights.testing.*`, how about
* `applicationinsights.testing.statsbeat.ikey`
* `applicationinsights.testing.statsbeat.endpoint`

Reply:
also, looks like we could simplify a bit and make StatsbeatConnectionString.create() method only take a single param, and handle the system property inside that method

Reply:
yes, that will need to be changed in the standalone exporter.

Reply:
maybe don't change it yet.. since it's only for smoke test?